src/olympia/promoted/tests/test_admin.py

```diff
@@ -44,7 +62,8 @@ def test_can_see_in_admin_with_discovery_edit(self):
         assert self.list_url in response.content.decode('utf-8')
 
     def test_can_list_with_discovery_edit_permission(self):
-        PromotedAddon.objects.create(addon=addon_factory(name='FooBâr'))
+        pro = PromotedAddon.objects.create(addon=addon_factory(name='FooBâr'))
+        PrimaryHero.objects.create(promoted_addon=pro)
```

Comment:
Why is this needed when the test does nothing afterwards related to the `PrimaryHero`?

Reply:
https://github.com/mozilla/addons-server/pull/15078/commits/c3759260619cb9d23bded23584fb801b2834f92c